packages/7zip-15-05.vm/tools/chocolateyinstall.ps1

```diff
@@ -30,11 +30,21 @@ try {
   $executablePath = Join-Path $toolDir "${toolName}FM.exe" -Resolve
   VM-Install-Shortcut $toolName $category $executablePath
 
+  # Test if the passowrd is right before extracting the files requiring two 7z commands with
+  # the two options `t` and `x`.
+  $7zCommand = @'
+	cmd /c (7z t -pinfected "%1" || (
+		mshta vbscript:Execute^(
+			"msgbox ""The password is not """"infected"""" or the archive is corrupted !!"":close"
+		^) && call
+	)) && 7z x -pinfected "%1"
+'@ -replace "`t", "" -replace "`n", ""
```

Comment:
Cool idea to make the code more readable! The alternative would be to have the code in a file. But as it is a small command, I think this is good too. :+1: